src/main/charts/jira/values.yaml

```diff
@@ -1,48 +1,198 @@
-# -- The initial number of pods that should be started at deployment of Jira.
-# Note that because Jira requires initial manual configuration after the first pod is deployed, and before scaling
-# up to additional pods, this should always be kept as 1.
+# -- The initial number of Jira pods that should be started at deployment time. 
+# Note: Jira requires manual configuration via the browser after the first pod is 
+# deployed. This configuration must be completed before scaling up additional pods. 
+# As such this value should always be kept as 1.
```

Comment:
The could change it to 2 after installation and do `helm upgrade`, but addressing that might be getting into the weeds a bit.